tests/support/foo-resolve-module-id/NonRel.ts

```diff
@@ -0,0 +1,5 @@
+﻿export class NonRel {
```

Comment:
I'm going to be picky here and say that we should call this NonRelative throughout, given that Rel can have multiple meanings.